src/main/java/uk/ac/ebi/eva/submission/service/WebinTokenService.java

```diff
@@ -10,7 +10,7 @@ public class WebinTokenService {
     private String userInfoUrl;
 
     public String getWebinUserIdFromToken(String userToken) {
-        String tokenAttribute = "id";
+        String tokenAttribute = "submissionAccountId";
```

Comment:
Is this due to a change in the webin REST response? May be I am misremembering it but I recall the id attribute worked the last time...